### ISSUE TITLE
Timber\Twig_Function not found

### ISSUE THREAD

**Issue by merijnponzo** [timber/timber#2318]
Hi,
On the staging server I got this error `Timber\Twig_Function` not found
I changed :
vendor/timber/timber/lib/Loader.php on line 436

to 
```
$twig->addFunction(new \Twig\TwigFunction('dump', function() {
```
With the help of 
```
Using the "Twig_Function" class is deprecated since Twig version 2.7, use "Twig\TwigFunction" instead
```

Thanks and good luck.
The multiple locations for .twig templates is the best improvement !

```
 Fatal error: Uncaught Error: Class 'Timber\Twig_Function' not found in /var/www/vhosts/belmondopetfood.nl/staging.belmondopetfood.nl/wp-content/themes/ponzotheme/app/vendor/timber/timber/lib/Loader.php on line 436
( ! ) Error: Class 'Timber\Twig_Function' not found in /var/www/vhosts/belmondopetfood.nl/staging.belmondopetfood.nl/wp-content/themes/ponzotheme/app/vendor/timber/timber/lib/Loader.php on line 436
Call Stack
#	Time	Memory	Function	Location
1	0.0005	366184	{main}( )	.../index.php:0
2	0.0007	366576	require( '/var/www/vhosts/belmondopetfood.nl/staging.belmondopetfood.nl/wp-blog-header.php' )	.../index.php:17
3	2.0541	6752024	require_once( '/var/www/vhosts/belmondopetfood.nl/staging.belmondopetfood.nl/wp-includes/template-loader.php' )	.../wp-blog-header.php:19
4	2.0577	6838288	include( '/var/www/vhosts/belmondopetfood.nl/staging.belmondopetfood.nl/wp-content/themes/ponzotheme/app/page.php' )	.../template-loader.php:106
5	2.0899	7086192	Timber\Timber::render( )	.../page.php:16
6	2.0913	7153216	Timber\Timber::compile( )	.../Timber.php:924
7	2.0953	7160032	Timber\Loader->render( )	.../Timber.php:789
8	2.0953	7160032	Timber\Loader->get_twig( )	.../Loader.php:105
```

**Comment by merijnponzo** [comment]
Hi Sorry for the late response
The webserver is using php 7.4
I installed timber with composer
```
{
    "require": {
        "timber/timber": "2.x-dev"
    },
    "minimum-stability": "dev"  
}
```
Wordpress 5.5.1

Within vendor/timber/timber/lib/Loader.php
Change line 436
```
$twig->addFunction(new Twig_Function('dump', function() {
```
to
```
$twig->addFunction(new \Twig\TwigFunction('dump', function() {
```
And the problem seem to be solved

Thank you

**Comment by jarednova** [comment]
This seems to be resolved by @acobster in #2327
